pkg/cmd/promhelperservice/handlers/instanceconfigs/delete.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 The Cockroach Authors.
+
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package instanceconfigs
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/cockroachdb/cockroach/pkg/cmd/promhelperservice/logging"
+)
+
+// DeleteHandler handles DELETE /instance-configs
+// Doc reference - https://cockroachlabs.atlassian.net/wiki/x/MAAlzg
+func DeleteHandler(w http.ResponseWriter, r *http.Request) {
+	clusterID := r.PathValue(ClusterIDParam)
```

Comment:
Can we follow the same pattern as in `PostHandler` where we call:

```
	requestBody, err := getInstanceConfigRequest(r.Body)
  clusterOD
```
To get the clusterID?

Reply:
In case of POST the cluster ID, which I am considering as the resource ID, is a part of the body. For PUT, DELETE, PATCH, it is a URL parameter.

Reply:
Actually, it makes sense to make the request a PUT. Reason being, the configuration is modified or created and the resource "clusterID" is already known. Will change it to PUT.